### PR TITLE
Fix sending search snowplow events

### PR DESCRIPTION
**Pull request by ranquild**
Related to https://github.com/metabase/metabase/pull/31467

There was an issue in the schema and the event was never tracked. This PR fixes the issue and adds e2e tests.

### CODE REVIEW

**Review by NevRA — APPROVED** [review]
LGTM

File: e2e/test/scenarios/onboarding/search/search.cy.spec.js [hunk 1]
```diff
@@ -63,6 +70,28 @@ describe("scenarios > auth > search", () => {
   });
 });
 
+describeWithSnowplow.only("scenarios > search", () => {
```
Comment (oisincoveney) [inline e2e/test/scenarios/onboarding/search/search.cy.spec.js:73]: Maybe need to remove this

```suggestion
describeWithSnowplow("scenarios > search", () => {
```

**Review by oisincoveney — APPROVED** [review]
Looks good, just the extra `only` within the test

File: e2e/test/scenarios/onboarding/search/search.cy.spec.js (hunk 1 above)
Comment (ranquild) [reply e2e/test/scenarios/onboarding/search/search.cy.spec.js]: Fixed

**Pull request merged**